pytorch_lightning/core/lightning.py

```diff
@@ -1588,7 +1603,7 @@ def get_progress_bar_dict(self) -> Dict[str, Union[int, str]]:
             tqdm_dict['split_idx'] = self.trainer.split_idx
 
         if self.trainer.logger is not None and self.trainer.logger.version is not None:
-            tqdm_dict['v_num'] = self.trainer.logger.version
+            tqdm_dict['v_num'] = str(self.trainer.logger.version)[:3]
```

Comment:
btw, why just three?

Reply:
github doesn't show it, but I pushed more commits to the branch. 
github has some issues at the moment it seems.

Reply:
> btw, why just three?

What would be a better number in your optinion? 5?

Reply:
I am just trying to understand why not all string...

Reply:
because some loggers like MLFlow use a very long string as a version and william suggested [here](https://github.com/PyTorchLightning/pytorch-lightning/issues/1595#issuecomment-619385389) to only show the last few bits in the progress bar.

Reply:
ok, but you show the first three here... :] so `str(self.trainer.logger.version)[-3:]`
so what about 7 as git does? :]

Reply:
> ok, but you show the first three here... :] so str(self.trainer.logger.version)[-3:]

yes I have that see latest commit. 
Let me ask on the original issue what people say about length